services/checks/checks_test.go

```diff
@@ -193,6 +193,22 @@ uEF33ScMPYpvHvBKv8+yBkJ9k4+DCfV4nDs6kKYwGhalvkkqwWkyfJffO+KW7a1m3y42WHpOnzBxLJ+I
 	})
 }
 
+func TestGetSecurityCheckResults(t *testing.T) {
+	t.Run("Check results not found error", func(t *testing.T) {
+		sqlDB := testdb.Open(t, models.SkipFixtures, nil)
+		db := reform.NewDB(sqlDB, postgresql.Dialect, nil)
+
+		defer func() {
+			require.NoError(t, sqlDB.Close())
```

Comment:
We can avoid that in the new code – `testdb.Open` closes connection in `Cleanup`

Reply:
not addressed yet